docker-for-aws/persistent-data-volumes.md

```diff
@@ -0,0 +1,45 @@
+---
+description: Persistent Data Volumes
+keywords: aws persistent data volumes
+title: Docker for AWS Persistent Data Volumes
+---
+
+## What is Cloudstor?
+Cloudstor is a Docker managed volume plugin that comes pre-installed (and pre-configured) in swarms deployed on Docker for AWS. Using volumes created through Cloudstor, swarm tasks can mount a persistent data volume that stays attached to the swarm tasks no matter which swarm node they get scheduled or migrated to. Currently Cloudstor heavily relies on shared storage infrastructure provided by the cloud platform to allow swarm tasks to create/mount their persistent volumes on any node in the swarm. In a future release we will introduce support for direct attached storage to satisfy very low latency/high IOPs requirements.
```

Comment:
a volume plugin managed by Docker. It comes pre-installed and pre-configured in swarms deployed on Docker for AWS. Swarm tasks use a volume created through Cloudstor to mount a persistent...Cloudstor relies on shared storage infrastructure provided by AWS to allow swarm tasks to...